### PR TITLE
Admin send letter pop up

### PR DESCRIPTION
**Pull request by annq3012**
## Description
Create pop-up confirm send email
## Reference Requirements
- Product Backlog ID: PBI-3
- Task ID: TI-3.3

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: public/app/js/borrow.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+$('.btn-check').click(function () {
+  $id = $(this).attr('id');
+})
+$('.btn-confirm').click(function () {
+  $('#form-confirm-'+$id).submit();
+})
```
Comment (at-anhngo) [inline public/app/js/borrow.js]: cần 1 khoảng trống

**Review by ghost — CHANGES_REQUESTED** [review]
Check my comment

**Review by ghost — APPROVED** [review]

**Comment by at-anhngo** [comment]
LGTM

File: resources/views/backend/books/borrow.blade.php [hunk 2]
```diff
@@ -1,7 +1,34 @@
 @extends('backend.layouts.main')
 @section('title',__('borrow.title_borrow'))
 @section('content')
+<!-- Modal -->
```
Comment (at-huydq) [inline resources/views/backend/books/borrow.blade.php]: tạo file `modal_confirm` rồi bỏ vào thư mục `backend/books/partials` giúp anh

File: resources/views/backend/books/borrow.blade.php [hunk 3]
```diff
@@ -63,4 +95,6 @@
 
 </div>
 <!-- /.content-wrapper -->
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```
Comment (at-huydq) [inline resources/views/backend/books/borrow.blade.php]: remove

File: resources/views/backend/books/borrow.blade.php [hunk 4]
```diff
@@ -63,4 +95,6 @@
...
+<script src="{{ asset('app/js/borrow.js') }}"></script>
```
Comment (at-huydq) [inline resources/views/backend/books/borrow.blade.php]: tạo `yield` `script-custom` bên `Main` nhá

**Review by ghost — COMMENTED** [review]
please check my comment

File: routes/web.php [hunk 5]
```diff
@@ -25,7 +25,7 @@
     Route::resource('borrowings', 'BorrowingController');
     Route::resource('categories', 'CategoryController');
     //Mail
-    Route::get('mail', 'Admin\SendMailController@sendMail')->name('sendMail');
+    Route::get('mail', 'SendMailController@sendMail')->name('sendMail');
```
Comment (at-huydq) [inline routes/web.php]: Send Mail là Method Post nhé

File: resources/views/backend/books/borrow.blade.php [hunk 6]
```diff
@@ -41,7 +45,12 @@
                                 <td>{{ date(config('define.date_format'), strtotime($borrowing->to_date)) }}</td>
                                 <td>{{ $borrowing->date_send_email }}</td>
                                 <td>
-                                    <a href="{{ route('sendMail') }}" id="{{ $borrowing->id }}" class="btn btn-warning"><i class="ion ion-android-drafts"></i></a>
+                                    <form method="GET" action="{{ route('sendMail') }}" id="form-confirm-{{$borrowing->id}}">
```
Comment (at-huydq) [inline resources/views/backend/books/borrow.blade.php]: `Post` not `Get`

**Comment by at-huydq** [comment]
https://media0.giphy.com/media/3osxYdek8wYWCOLgT6/200_d.webp?cid=cafe52e95a6ffda55a3656412edf66fc
LGTM

**Pull request merged**